scripts/windows-native-ci.ps1

```
@@ -4207,8 +4296,12 @@ function Invoke-WindowsReleaseCertification {
     if ([string]$provenance.version -cne $releaseVersion) {
         throw "release setup provenance version does not match the resolved release: $($provenance.version) != $releaseVersion"
     }
+    $expectedHookLauncherHash = [string]$provenance.inputs.hook_launcher_sha256
+    if ($expectedHookLauncherHash -cnotmatch '^[0-9a-f]{64}$') {
+        throw 'release setup provenance lacks the canonical HookRuntime launcher SHA-256'
+    }
     Assert-WindowsReleaseSbom `
-        $sbomPath $setupHash $releaseVersion ([string]$env:GITHUB_SHA)
+        $sbomPath $setupHash $releaseVersion ([string]$env:GITHUB_SHA) $expectedHookLauncherHash
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Assert-WindowsReleaseSbom' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)